tests/app/run.sh

```diff
@@ -37,5 +37,6 @@ artifact_file=/tests/$(basename "$artifact_file")
 
 while read -r; do
     echo "running $REPLY"
-    EXPECTED_CONTAINERS="${root_dir}/data/docker"/expected.containers-delta.txt ARTIFACT_FILE="${artifact_file}" DATA="${root_dir}/data/docker" MODULE="${module}" "$REPLY" && echo -e "$REPLY \E[0;32mPASSED\E[0m"
+    EXPECTED_CONTAINERS="${root_dir}/data/docker"/expected.containers-delta.txt ARTIFACT_FILE="${artifact_file}" DATA="${root_dir}/data/docker" MODULE="${module}" "$REPLY" || exit 1
```

Comment:
Q:
 Why are all these on a single line, with only a space separating them?
 :thinking: 
Is it to feed into the command? 
I think I would still prefer to define them each on a separate line, and then feed them to the command, this was a little long to decipher.
Maybe at least a multi-line command :pray:

Reply:
I thought it would be a good idea to pass the variables in the `V=value command_using_variable_V` in this case. I moved to the multiline command now.